conda_build/cli/main_index.py

```diff
@@ -90,8 +90,8 @@ def execute(args):
 
     api.update_index(args.dir, check_md5=args.check_md5, channel_name=args.channel_name,
```

Comment:
Would love to delete this whole file.

`conda-index` has a nice click CLI, but doesn't take the same arguments.

Reply:
this might be able to use the non-api update_index, which is basically "call update_index for every dir in a list"